suricata/package.yaml

```diff
@@ -1,33 +1,1226 @@
+---
 id: suricata
 name: Suricata
 author: Tenzir
 author_icon: https://github.com/tenzir.png
-package_icon: https://suricata.io/wp-content/uploads/2023/09/Logo-Suricata-vert-R.png
-
+package_icon: |
+  https://suricata.io/wp-content/uploads/2023/09/Logo-Suricata-vert-R.png
 description: |
   A package to simplify the onboarding of logs emitted from a Suricata
   instance.
-
 inputs:
   eve-path:
     name: EVE Path
     description: A path where suricata writes eve.json formatted logs
     type: string
     default: /var/lib/suricata/eve.sock
-
+  eve-address:
+    name: EVE Address
+    description: A network address that suricata forwards formatted logs to
+    type: string
+    default: tcp://0.0.0.0:0000
 pipelines:
-  import:
-    name: Suricata Import
-    description: Ingest Suricata events into the internal storage of the node.
+  publish-from-file:
+    name: Suricata Publish
+    description: Ingest Suricata events from a fileinto a subscription topic.
     restart-on-error: 5m
+    disabled: false
     definition: |
+      // tql2
       from file {{ inputs.eve-path }} --follow read suricata
-      | import
-
+      | publish suricata
+  publish-from-network:
+    name: Suricata Publish
+    description: |
+      Ingest Suricata events from a network source into a subscription topic.
+    restart-on-error: 5m
+    disabled: true
+    definition: |
+      // tql2
+      from {{ inputs.eve-address }} read suricata
+      | buffer 16Mi --policy drop
+      | publish suricata
```

Comment:
This mixes TQL1 and TQL2.

Reply:
Yes, those were remnants of the previous package version.
I created TQL2 pipelines for those in https://github.com/tenzir/library/pull/5/commits/c49583aa3ff53da4a63a046566c2126237252d7a.